runtime/src/xcm_config.rs

```diff
@@ -289,9 +242,73 @@ impl orml_xtokens::Config for Runtime {
 	type XcmExecutor = XcmExecutor<XcmConfig>;
 	type Weigher = FixedWeightBounds<UnitWeightCost, RuntimeCall, MaxInstructions>;
 	type BaseXcmWeight = BaseXcmWeight;
-	type LocationInverter = LocationInverter<Ancestry>;
 	type MaxAssetsForTransfer = MaxAssetsForTransfer;
 	type MinXcmFee = ParachainMinFee;
 	type MultiLocationsFilter = Everything;
 	type ReserveProvider = AbsoluteReserveProvider;
+	type UniversalLocation = UniversalLocation;
+}
+
+// The pallet will be disabled for extarnal calls
+pub struct PalletXCMWeightInfo;
+impl pallet_xcm::WeightInfo for PalletXCMWeightInfo {
+	fn send() -> XcmWeight {
+		XcmWeight::from_ref_time(0)
```

Comment:
`XcmWeight::zero()` if this method exists

Reply:
Done